docs/design/jet/022-mongodb-connector.md

```diff
@@ -121,10 +121,8 @@ p.readFrom(Sources.list(list))
 User should be able to run following SQL query:
 ```sql
 CREATE MAPPING people
-TYPE MONGODB 
-OPTIONS (
-  'dataLinkRef'='mongodb-ref' 
-)
+DATA CONNECTION "mongodb-ref"
+TYPE MONGODB
```

Comment:
type is redundant now, maybe worth removing where possible

Reply:
5b46977cc6f42a0b4990acf291c842fabd7a1432